app/home/components/mailingSection/mailingSection.scss

```diff
@@ -1,88 +1,100 @@
 $container-width: 1080px;
 
 .mailingSectionContainer {
-  padding-top: 50px;
-  padding-bottom: 130px;
-
+  padding-top: 62px;
+  padding-bottom: 58px;
+  background-color: #6096ff;
   .innerContainer {
     max-width: $container-width;
-    margin: 0 auto;
-  }
-
-  @media (max-width: 768px) {
-    padding-bottom: 80px;
-    .innerContainer {
-      padding-left: 15px;
-      padding-right: 15px;
-    }
-  }
-}
-
-.title {
-  font-size: 28px;
-  line-height: 1.7;
-  text-align: center;
-  color: #172649;
-  margin-bottom: 45px;
-
-  strong {
-    font-weight: 700;
-  }
-
-  @media (max-width: 768px) {
-    font-size: 18px;
-    line-height: 1.56;
-  }
-}
-
-.emailFormWrapper {
-  text-align: center;
-
-  .emailSubmitInput {
-    display: inline-block;
-    vertical-align: top;
-    width: 310px;
-    height: 45px;
-    border-radius: 9px;
-    -webkit-backdrop-filter: blur(9px);
-    background-color: rgba(255, 255, 255, 0.7);
-    border: solid 1px #6096ff;
-    padding-left: 20px;
-    padding-right: 20px;
-    &:focus {
-      outline: none;
+    margin-left: auto;
+    margin-right: auto;
+    display: flex;
+    flex-direction: row;
+    .ctaWrapper {
```

Comment:
Can you give a space per separating class?

Reply:
.someClass
  fdsjklfdsjfkdlsjfdksjfds
  dsfjldfksjfdskljfds

  .theOtherClass

would be more easy to read